lib/charms/mysql/v0/mysql.py

```diff
@@ -2434,7 +2550,7 @@ def restart_mysql_exporter(self) -> None:
         raise NotImplementedError
 
     @abstractmethod
-    def wait_until_mysql_connection(self) -> None:
+    def wait_until_mysql_connection(self, check_port: bool = True) -> None:
```

Comment:
because of this change, the implementation of `wait_until_mysql_connection` in mysql_k8s_helpers will need to take an additional kwarg `check_port`

Reply:
be4f0c8